### ISSUE TITLE
searchApi.getSearchHints indicates an error when SearchHint.matchedTerm is null

### ISSUE THREAD

**Issue by xianyvbang** [jellyfin/jellyfin-sdk-kotlin#936]
Version:
```
<dependency>
                <groupId>org.jellyfin.sdk</groupId>
                <artifactId>jellyfin-core-jvm</artifactId>
                <version>1.5.0-beta.1</version>
            </dependency>
```

``` kotlin
fun musicSearchHint(search: String): SearchHintResult {
        val pageNumber = Convert.toInt(ServletUtils.getParameter(TableSupport.PAGE_NUM), 1)
        val pageSize = Convert.toInt(ServletUtils.getParameter(TableSupport.PAGE_SIZE), 10)
        return runBlocking {
            apiClient.searchApi.getSearchHints(
                limit = pageSize,
                searchTerm = search,
                startIndex = (pageNumber - 1) * pageSize,
                includeItemTypes = listOf(
                    BaseItemKind.AUDIO, BaseItemKind.AUDIO_BOOK,
                    BaseItemKind.MUSIC_ALBUM, BaseItemKind.MUSIC_ARTIST
                )
            )
        }.content
    }
```

```
Caused by: kotlinx.serialization.MissingFieldException: Field 'MatchedTerm' is required for type with serial name 'org.jellyfin.sdk.model.api.SearchHint', but it was missing at path: $.SearchHints[0]
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:95)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
	at kotlinx.serialization.encoding.CompositeDecoder$DefaultImpls.decodeSerializableElement$default(Decoding.kt:538)
	at kotlinx.serialization.internal.CollectionLikeSerializer.readElement(CollectionSerializers.kt:80)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.readElement$default(CollectionSerializers.kt:51)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.merge(CollectionSerializers.kt:36)
	at kotlinx.serialization.internal.AbstractCollectionSerializer.deserialize(CollectionSerializers.kt:43)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableElement(AbstractDecoder.kt:70)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableElement(StreamingJsonDecoder.kt:168)
	at org.jellyfin.sdk.model.api.SearchHintResult$$serializer.deserialize(SearchHintResult.kt:16)
	at org.jellyfin.sdk.model.api.SearchHintResult$$serializer.deserialize(SearchHintResult.kt:16)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
	at kotlinx.serialization.json.Json.decodeFromString(Json.kt:107)
	at org.jellyfin.sdk.api.operations.SearchApi.getSearchHints(SearchApi.kt:145)
	... 128 common frames omitted
Caused by: kotlinx.serialization.MissingFieldException: Field 'MatchedTerm' is required for type with serial name 'org.jellyfin.sdk.model.api.SearchHint', but it was missing
	at kotlinx.serialization.internal.PluginExceptionsKt.throwMissingFieldException(PluginExceptions.kt:20)
	at org.jellyfin.sdk.model.api.SearchHint.<init>(SearchHint.kt:31)
	at org.jellyfin.sdk.model.api.SearchHint$$serializer.deserialize(SearchHint.kt:31)
	at org.jellyfin.sdk.model.api.SearchHint$$serializer.deserialize(SearchHint.kt:31)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
	... 145 common frames omitted
```

**Comment by nielsvanvelzen** [comment]
What is the version of the Jellyfin server you tested with?

**Comment by xianyvbang** [comment]
> What is the version of the Jellyfin server you tested with?

10.9.2

**Comment by nielsvanvelzen** [comment]
The issue was in the server causing the OpenAPI scheme to be incorrect. It will be fixed with 10.9.3 (https://github.com/jellyfin/jellyfin/pull/11802), so next beta of the SDK should work again.

**Comment by xianyvbang** [comment]
> The issue was in the server causing the OpenAPI scheme to be incorrect. It will be fixed with 10.9.3 ([jellyfin/jellyfin#11802](https://github.com/jellyfin/jellyfin/pull/11802)), so next beta of the SDK should work again.

Okay, thanks
